cfme/cloud/provider.py

```diff
@@ -15,23 +15,18 @@
 
 import cfme
 import cfme.fixtures.pytest_selenium as sel
-from cfme.web_ui import flash
 from cfme.web_ui import form_buttons
 from cfme.web_ui import toolbar as tb
 from cfme.common.provider import BaseProvider
 import cfme.web_ui.menu  # so that menu is already loaded before grafting onto it
```

Comment:
If we import ```nav``` from the ```menu``` module and not from the ```ui_navigate``` module, won't that mean that we will get what we need and lost the lint error?